Makefile

```diff
@@ -60,11 +63,13 @@ version-check: .common-pkgs.txt apro-abi@$(APRO_VERSION).pkgs.txt
 	}
 .PHONY: version-check
 
-%.so: %.go download-go download-docker version-check
+%.so: %.go download-go download-docker version-check sync
 	$(go.GOBUILD) -buildmode=plugin -o $@ $<
+	rsync -e 'docker exec -i' -r $(shell docker ps -q -f label=component=plugin-builder):${CURDIR}/ .
```

Comment:
This could make use of the `$(container.ID)` variable.

Reply:
Good point. It wasn't working when I first tried it for some reason. Working now.